packages/element-lib/src/sidetree-v2/index.js

```diff
@@ -0,0 +1,14 @@
+const { resolve, sync, create } = require('./protocol');
+
+class Sidetree {
+  constructor({ db, blockchain, storage }) {
+    this.blockchain = blockchain;
+    this.storage = storage;
+    this.db = db;
+    this.resolve = resolve(this);
```

Comment:
this kind of registration breaks vscode tooling support / making it harder to actually use this library... another reason to switch to typescript.